src/Abp/BackgroundJobs/BackgroundJobManagerEventTriggerExtensions.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Threading.Tasks;
+using Abp.Events.Bus;
+
+namespace Abp.BackgroundJobs
+{
+    public static class BackgroundJobManagerEventTriggerExtensions
+    {
+        
```

Comment:
* Clean unnecessary empty lines
* Rename T to TEvent in the EnqueueEventAsync method and "IBackgroundJobManager t" to "IBackgroundJobManager backgroundJobManager".